test_system.TestKMSIntegration.test_bucket_w_default_kms_key_name failed

Got a flaky error on kokoro checks. Failed only once, on Python 2.7 session. Trying to reproduce locally - no results so far.

```python
____________ TestKMSIntegration.test_bucket_w_default_kms_key_name _____________

self = <test_system.TestKMSIntegration testMethod=test_bucket_w_default_kms_key_name>

    def test_bucket_w_default_kms_key_name(self):
        BLOB_NAME = "default-kms-key-name"
        OVERRIDE_BLOB_NAME = "override-default-kms-key-name"
        ALT_BLOB_NAME = "alt-default-kms-key-name"
        CLEARTEXT_BLOB_NAME = "cleartext"

        file_data = self.FILES["simple"]

        with open(file_data["path"], "rb") as _file_data:
            contents = _file_data.read()

        kms_key_name = self._kms_key_name()
        self.bucket.default_kms_key_name = kms_key_name
        self.bucket.patch()
        self.assertEqual(self.bucket.default_kms_key_name, kms_key_name)

        defaulted_blob = self.bucket.blob(BLOB_NAME)
        defaulted_blob.upload_from_filename(file_data["path"])
        self.case_blobs_to_delete.append(defaulted_blob)

        self.assertEqual(defaulted_blob.download_as_string(), contents)
        # We don't know the current version of the key.
        self.assertTrue(defaulted_blob.kms_key_name.startswith(kms_key_name))

        alt_kms_key_name = self._kms_key_name(self.ALT_KEY_NAME)

        override_blob = self.bucket.blob(
            OVERRIDE_BLOB_NAME, kms_key_name=alt_kms_key_name
        )
        override_blob.upload_from_filename(file_data["path"])
        self.case_blobs_to_delete.append(override_blob)

        self.assertEqual(override_blob.download_as_string(), contents)
        # We don't know the current version of the key.
        self.assertTrue(override_blob.kms_key_name.startswith(alt_kms_key_name))

        self.bucket.default_kms_key_name = alt_kms_key_name
        self.bucket.patch()

        alt_blob = self.bucket.blob(ALT_BLOB_NAME)
        alt_blob.upload_from_filename(file_data["path"])
        self.case_blobs_to_delete.append(alt_blob)

        self.assertEqual(alt_blob.download_as_string(), contents)
        # We don't know the current version of the key.
>       self.assertTrue(alt_blob.kms_key_name.startswith(alt_kms_key_name))
E       AssertionError: False is not true
```

--- a/tests/system/test_system.py
+++ b/tests/system/test_system.py
@@ -1905,6 +1905,7 @@ class TestKMSIntegration(TestStorageFiles):
         (listed,) = list(self.bucket.list_blobs())
         self.assertTrue(listed.kms_key_name.startswith(kms_key_name))
 
+    @RetryErrors(unittest.TestCase.failureException)
     def test_bucket_w_default_kms_key_name(self):
         BLOB_NAME = "default-kms-key-name"
         OVERRIDE_BLOB_NAME = "override-default-kms-key-name"

